test/layouts/test_wmii.py

```diff
@@ -61,9 +61,9 @@ def test_wmii_window_focus_cycle(qtile):
     qtile.testWindow("three")
 
     # test preconditions
-    assert qtile.c.layout.info()['clients'] == ['one', 'two', 'three']
+    assert qtile.c.layout.info()['clients'] == ['three', 'two', 'one']
```

Comment:
So I guess the difference that we capture in our tests is that `Columns` inserts new clients above, while `Wmii` inserts new clients below. It looks like the the `_ClientList.add` has an `offset_to_current` arg that could be used to account for this. Do you think it would be possible to add that as a parameter that is used, so `Wmii` behavior doesn't change?